GoSSB/Sources/api-ios.go

```diff
@@ -315,6 +318,8 @@ func ssbBotInit(config string, notifyBlobReceivedFn uintptr, notifyNewBearerToke
 		}),
 		mksbot.DisableEBT(true),
 		mksbot.WithPublicAuthorizer(newAcceptAllAuthorizer()),
+		mksbot.WithNumberOfConcurrentReplicationsPerPeer(25),
```

Comment:
I see that we went down to 25, I think that is reasonable.

Reply:
Yeah given that we are currently bottlenecked by SQLite I am trying to make replication a little slower which gives SQLite a few more resources. I may tweak it again if I can get significantly more performance out of SQLite.

Btw I think these options would be great to contribute back to cryptoscope/ssb in some way if it makes sense. It gives anyone using this as a library some coarse control over memory usage which seems important on any device that isn't a modern desktop.